[automation] Startlevel payload in automation rules always set to level 40 after saving rule.

I noticed a strange behaviour with the system startup level triggers after saving an automation rule. (e.g. JSScripting and Python)

I've created a rule that triggers on every system startup level with JSScripting. When OpenHab starts up this rule works as expected and is triggered for every startup level with the correct startlevel as payload. But when the file of the rule is changed or saved again the rule is triggered directly after saving and always/only with startlevel 40 as payload. The startlevel 40 is also triggered if I only configure a trigger for startup level 100.

If I use the DSL rules I get an event for every startup level in order after saving the rules, but I have to create a separate rule for every startup level because there is no startlevel payload.

This is a problem for me because I'm using Typescript to write the JSScripting rules and because of the Typescript generation the files get overwritten every time I change something in any rule.

I also started a discussion in the forum about this and was advised to create an issue for this: https://community.openhab.org/t/systemstartleveltrigger-in-javascript

## Expected Behavior
After saving an automation rule with system startup level triggers it should trigger with the correct startup level in the payload. Or it should trigger with every startup level as payload in order just like with the DSL rules.

## Current Behavior
After saving an automation rule with startup level triggers (with JSScripting or Python) it is immediateley triggered with and only with startup level 40.

## Steps to Reproduce (for Bugs)

1. Create the JSScripting or Python rule as shown below.
2. Look at the debug output of the rule. After saving it is always only getting startup level 40 in the payload.

### JSScripting Rule
```
configuration: {}
triggers:
  - id: "1"
    configuration:
      startlevel: 20
    type: core.SystemStartlevelTrigger
  - id: "2"
    configuration:
      startlevel: 40
    type: core.SystemStartlevelTrigger
  - id: "3"
    configuration:
      startlevel: 50
    type: core.SystemStartlevelTrigger
  - id: "4"
    configuration:
      startlevel: 70
    type: core.SystemStartlevelTrigger
  - id: "5"
    configuration:
      startlevel: 80
    type: core.SystemStartlevelTrigger
  - id: "6"
    configuration:
      startlevel: 100
    type: core.SystemStartlevelTrigger
conditions: []
actions:
  - inputs: {}
    id: "7"
    configuration:
      type: application/javascript
      script: >-
        var strLevel = "";

        switch (startlevel.toString()) {
          case '20':
            strLevel = "(System Started)";
            break;
          case '40':
            strLevel = "(Rules are loaded)";
            break;
          case '50':
            strLevel = "(Rules Engine active)";
            break;
          case '70':
            strLevel = "(UI is active)";
            break;
          case '80':
            strLevel = "(All things initialized)";
            break;
          case '100':
            strLevel = "(Startup complete)";
        } 

        console.error("System Start Level:",startlevel.toString(),strLevel,event);
    type: script.ScriptAction
```
### Python Rule
```
from openhab import rule
from openhab.triggers import SystemStartlevelTrigger

@rule(
    triggers=[
        SystemStartlevelTrigger(20),
        SystemStartlevelTrigger(40),
        SystemStartlevelTrigger(50),
        SystemStartlevelTrigger(70),
        SystemStartlevelTrigger(80),
        SystemStartlevelTrigger(100),
    ]
)
class Test:
    def execute(self, module, input):
        for key, value in input.items():
            self.logger.error(f"{key} = {value}")
```

## Context
This is a problem for me because I'm using Typescript to write the JSScripting rules and because of the Typescript generation the files get overwritten every time I change something in any rule. And I use this rule to track the current startup level and only send alerts when startup level 100 is reached. So this always deactivates all my alert rules.

## Your Environment
* Version used: openHAB 5.0.3 (I also tested this with openHAB 5.1 and got the same result)
* Environment name and version: Rasperry Pi 5
* Operating System and version: Rasperry Pi OS (Debian 12)

> I'm sure you can find workarounds for most things, that more or less works. My example was perhaps explained badly, but I think there's a clear difference between "an init procedure that must be executed before a set of rules can do what they do" and "an init procedure that must be executed once as soon as possible when the system starts, and only then".

I have another problem. The rules are triggered twice. I'm doing nothing. On different machines. 
Moreover, as you can see, the Telegram binding is unavailable in one rule (rule system start = 100) , but I check everything (null and so on). 
And in the other, there's an error with the item, even though it exists during normal operation. 
I don't know how to debug this. It started with version 4.X.

There may or may not be errors during the restart. Everything is unstable.

```
2025-12-23 12:06:18.097 [INFO ] [org.openhab.core.Activator          ] - Starting openHAB 5.1.0 (Release Build)
2025-12-23 12:06:38.125 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'debug_sensor_climat_voltage.items'
2025-12-23 12:06:39.125 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'jdbc.persist'
2025-12-23 12:06:39.613 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'mapdb.persist'
2025-12-23 12:06:48.649 [INFO ] [.core.model.lsp.internal.ModelServer] - Started Language Server Protocol (LSP) service on port 5007
2025-12-23 12:06:49.293 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'openhab.rules'
2025-12-23 12:06:53.009 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'intercom.rules'
2025-12-23 12:06:53.782 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'rflnk.rules'
2025-12-23 12:06:58.730 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'socketplug.rules'
2025-12-23 12:07:02.354 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'bluetooth.rules'
2025-12-23 12:07:03.045 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'lightbulb.rules'
2025-12-23 12:07:04.872 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'relay.rules'
2025-12-23 12:07:05.481 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'debug.rules'
2025-12-23 12:07:07.091 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'siren.rules'
2025-12-23 12:07:08.442 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'weather.rules'
2025-12-23 12:07:08.630 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'intrusion.rules'
2025-12-23 12:07:12.837 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'alarms.rules'
2025-12-23 12:07:22.030 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'voice.rules'
2025-12-23 12:07:31.152 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'network.rules'
2025-12-23 12:07:33.801 [INFO ] [el.core.internal.ModelRepositoryImpl] - Loading DSL model 'scenes.rules'
2025-12-23 12:08:05.566 [INFO ] [hab.io.metrics.MetricsRestController] - Metrics service activated, serving the following URL(s): /rest/metrics/prometheus
2025-12-23 12:08:05.811 [INFO ] [persistence.jdbc.internal.JdbcMapper] - JDBC::openConnection: Driver is available::Yank setupDataSource
2025-12-23 12:08:21.378 [INFO ] [ab.ui.habpanel.internal.HABPanelTile] - Started HABPanel at /habpanel
2025-12-23 12:08:34.346 [INFO ] [.transport.mqtt.MqttBrokerConnection] - Starting MQTT broker connection to '127.0.0.1' with clientid openhab
2025-12-23 12:08:42.332 [WARN ] [core.thing.internal.ThingManagerImpl] - Initializing handler for thing 'mqtt:topic:mosquitto:zanussi_zac_dev01' takes more than 5000ms.
2025-12-23 12:08:43.742 [INFO ] [e.automation.internal.RuleEngineImpl] - Rule engine started.
2025-12-23 12:08:51.025 [INFO ] [ab.core.model.script.lightbulb_rules] - Initialization 'lightbulb.rules'
2025-12-23 12:08:51.025 [INFO ] [enhab.core.model.script.rflink_rules] - Initialization 'rflink.rules'
2025-12-23 12:08:51.026 [INFO ] [penhab.core.model.script.siren_rules] - Initialization 'siren.rules'
2025-12-23 12:08:51.027 [INFO ] [ab.core.model.script.intrusion_rules] - Initialization 'intrusion.rules'
2025-12-23 12:08:51.027 [INFO ] [enhab.core.model.script.alarms_rules] - Initialization 'alarms.rules'
2025-12-23 12:08:51.027 [INFO ] [nhab.core.model.script.openhab_rules] - Initialization 'openhab.rules'
2025-12-23 12:08:51.027 [INFO ] [b.core.model.script.socketplug_rules] - Initialization 'socketplug.rules'
2025-12-23 12:08:51.027 [INFO ] [penhab.core.model.script.debug_rules] - Initialization 'debug.rules'
2025-12-23 12:08:51.028 [INFO ] [nhab.core.model.script.network_rules] - Initialization 'network.rules'
2025-12-23 12:08:51.027 [INFO ] [ab.core.model.script.bluetooth_rules] - Initialization 'bluetooth.rules'
2025-12-23 12:08:51.029 [INFO ] [enhab.core.model.script.scenes_rules] - Initialization 'scenes.rules'
2025-12-23 12:08:51.030 [INFO ] [penhab.core.model.script.voice_rules] - Initialization 'voice.rules'
2025-12-23 12:08:51.090 [INFO ] [core.model.script.network_wol_device] - network:pingdevice:MADRAGE
2025-12-23 12:08:51.096 [ERROR] [.handler.AbstractScriptModuleHandler] - Script execution of rule with UID 'openhab-1' failed: 'sendTelegram' is not a member of 'org.openhab.core.thing.binding.ThingActions'; line 51, column 5, length 70 in openhab
2025-12-23 12:08:51.158 [INFO ] [core.model.script.sonoff_rfbr2_dev01] - state initialized
2025-12-23 12:09:19.924 [INFO ] [nhab.core.model.script.openhab_rules] - Initialization 'openhab.rules'
2025-12-23 12:09:22.447 [INFO ] [enhab.core.model.script.rflink_rules] - Initialization 'rflink.rules'
2025-12-23 12:09:24.295 [INFO ] [b.core.model.script.socketplug_rules] - Initialization 'socketplug.rules'
2025-12-23 12:09:24.749 [INFO ] [ab.core.model.script.bluetooth_rules] - Initialization 'bluetooth.rules'
2025-12-23 12:09:27.361 [INFO ] [ab.core.model.script.lightbulb_rules] - Initialization 'lightbulb.rules'
2025-12-23 12:09:28.254 [INFO ] [penhab.core.model.script.debug_rules] - Initialization 'debug.rules'
2025-12-23 12:09:30.390 [INFO ] [penhab.core.model.script.siren_rules] - Initialization 'siren.rules'
2025-12-23 12:09:32.964 [INFO ] [ab.core.model.script.intrusion_rules] - Initialization 'intrusion.rules'
2025-12-23 12:09:33.040 [ERROR] [.handler.AbstractScriptModuleHandler] - Script execution of rule with UID 'alarms-10' failed: The name 'alarm_alert_signal_low' cannot be resolved to an item or type; line 728, column 36, length 22 in alarms
2025-12-23 12:09:37.087 [INFO ] [enhab.core.model.script.alarms_rules] - Initialization 'alarms.rules'
2025-12-23 12:09:41.301 [INFO ] [penhab.core.model.script.voice_rules] - Initialization 'voice.rules'
2025-12-23 12:09:42.965 [INFO ] [nhab.core.model.script.network_rules] - Initialization 'network.rules'
2025-12-23 12:09:43.612 [INFO ] [enhab.core.model.script.scenes_rules] - Initialization 'scenes.rules'
```

--- a/bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/RuleEngineImpl.java
+++ b/bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/RuleEngineImpl.java
@@ -872,11 +872,12 @@ public class RuleEngineImpl implements RuleManager, RegistryChangeListener<Modul
         register(rule);
         setStatus(ruleUID, new RuleStatusInfo(RuleStatus.IDLE));
 
-        // check if we have to trigger because of the startlevel
+        // If the rule engine is started: Check if we have to trigger immediately because of the startlevel
+        // Execution of startlevel triggers during startup is handled by executeRulesWithStartlevel()
         List<Trigger> slTriggers = rule.getTriggers().stream().map(WrappedTrigger::unwrap)
                 .filter(t -> SystemTriggerHandler.STARTLEVEL_MODULE_TYPE_ID.equals(t.getTypeUID())).toList();
         int startLevel = startLevelService.getStartLevel();
-        if (slTriggers.stream()
+        if (started && slTriggers.stream()
                 .anyMatch(t -> ((BigDecimal) t.getConfiguration().get(SystemTriggerHandler.CFG_STARTLEVEL))
                         .intValue() <= startLevel)) {
             runNow(rule.getUID(), true, Map.of(SystemTriggerHandler.OUT_STARTLEVEL, StartLevelService.STARTLEVEL_RULES,

